go.mod

```diff
@@ -32,34 +25,29 @@ require (
 	github.com/ONSdigital/dp-net/v2 v2.11.2
 )
 
+require golang.org/x/sync v0.7.0 // indirect
```

Comment:
Why is this in it's own block?

Reply:
This is added automatically when running go get & go mod tidy.